app/controllers/patients_controller.rb

```diff
@@ -534,104 +519,110 @@ def patient_diff(patient_before, patient_after)
   def allowed_params
-    %i[
-      user_defined_id_statelocal
-      user_defined_id_cdc
-      user_defined_id_nndss
-      first_name
-      middle_name
-      last_name
-      date_of_birth
-      age
-      sex
-      white
-      black_or_african_american
-      american_indian_or_alaska_native
-      asian
-      native_hawaiian_or_other_pacific_islander
-      ethnicity
-      primary_language
-      secondary_language
-      interpretation_required
-      nationality
-      address_line_1
-      foreign_address_line_1
-      address_city
-      address_state
-      address_line_2
-      address_zip
-      address_county
-      monitored_address_line_1
-      monitored_address_city
-      monitored_address_state
-      monitored_address_line_2
-      monitored_address_zip
-      monitored_address_county
-      foreign_address_city
-      foreign_address_country
-      foreign_address_line_2
-      foreign_address_zip
-      foreign_address_line_3
-      foreign_address_state
-      foreign_monitored_address_line_1
-      foreign_monitored_address_city
-      foreign_monitored_address_state
-      foreign_monitored_address_line_2
-      foreign_monitored_address_zip
-      foreign_monitored_address_county
-      primary_telephone
-      primary_telephone_type
-      secondary_telephone
-      secondary_telephone_type
-      email
-      preferred_contact_method
-      preferred_contact_time
-      port_of_origin
-      source_of_report
-      source_of_report_specify
-      flight_or_vessel_number
-      flight_or_vessel_carrier
-      port_of_entry_into_usa
-      travel_related_notes
-      additional_planned_travel_type
-      additional_planned_travel_destination
-      additional_planned_travel_destination_state
-      additional_planned_travel_destination_country
-      additional_planned_travel_port_of_departure
-      date_of_departure
-      date_of_arrival
-      additional_planned_travel_start_date
-      additional_planned_travel_end_date
-      additional_planned_travel_related_notes
-      last_date_of_exposure
-      potential_exposure_location
-      potential_exposure_country
-      contact_of_known_case
-      contact_of_known_case_id
-      travel_to_affected_country_or_area
-      was_in_health_care_facility_with_known_cases
-      was_in_health_care_facility_with_known_cases_facility_name
-      laboratory_personnel
-      laboratory_personnel_facility_name
-      healthcare_personnel
-      healthcare_personnel_facility_name
-      exposure_notes
-      crew_on_passenger_or_cargo_flight
-      monitoring_plan
-      exposure_risk_assessment
-      member_of_a_common_exposure_cohort
-      member_of_a_common_exposure_cohort_type
-      isolation
-      jurisdiction_id
-      assigned_user
-      symptom_onset
-      extended_isolation
-      case_status
-      continuous_exposure
-      gender_identity
-      sexual_orientation
-      user_defined_symptom_onset
-    ]
+    params.require(:patient).permit(
```

Comment:
We need to make sure this change doesn't affect the use of `allowed_params` elsewhere. I can already see it would cause issues here: https://github.com/SaraAlert/SaraAlert/blob/master/app/controllers/patients_controller.rb#L515